src/components/License.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+
+import { type StacItem } from "../utils/stac.ts"
+import { computed } from "vue";
+
+const {item} = defineProps<{ item: StacItem }>();
+
+const licenseImage = computed(() =>
+  item?.properties?.license.startsWith('CC-BY-NC-SA') ? '/src/images/by-nc-sa.svg' :
```

Comment:
This approach with the absolute path to the license images will likely break when deployed to the production site: the site is deployed to a subdirectory, so the absolute path wouldn't reference the actual image. A better approach likely is to import the SVG as a component as in: https://github.com/orcestra-campaign/ipfsui/blob/b48dfcac22e1294a5fdc688bf836ed56425d17bd/src/components/IPFSStatus.vue#L4

This would not only solve the path issue, but also inline the svg in the bundled javascript, reducing the required amount of data transfers.